CMakeLists.txt

```diff
@@ -60,9 +60,95 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
   message(FATAL_ERROR "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there.\n")
 endif()
 
+#
+# Setup libbpf
+#
+set(LIBBPF_DIR ${CMAKE_CURRENT_SOURCE_DIR}/third_party/libbpf/)

```

Comment:
it would be better putting it in a seperate .cmake file.

Reply:
Fixed now